apps/designer/app/dashboard/project-card.tsx

```diff
@@ -0,0 +1,295 @@
+import {
+  DropdownMenu,
+  DropdownMenuTrigger,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  IconButton,
+  css,
+  Flex,
+  Text,
+  theme,
+  toast,
+} from "@webstudio-is/design-system";
+import { MenuIcon } from "@webstudio-is/icons";
+import type { DashboardProject } from "@webstudio-is/prisma-client";
+import { KeyboardEvent, useEffect, useRef, useState } from "react";
+import {
+  dashboardProjectPath,
+  designerPath,
+  getPublishedUrl,
+} from "~/shared/router-utils";
+import { Link as RemixLink, useFetcher } from "@remix-run/react";
+import { isFeatureEnabled } from "@webstudio-is/feature-flags";
+import { createTrpcRemixProxy } from "../shared/remix/create-trpc-remix-proxy";
+import type { DashboardProjectRouter } from "@webstudio-is/dashboard";
+
+const projectCardContainerStyle = css({
+  overflow: "hidden",
+  width: theme.spacing[31],
+  height: theme.spacing[29],
+  borderWidth: 1,
+  borderStyle: "solid",
+  borderColor: theme.colors.borderMain,
+  borderRadius: theme.borderRadius[4],
+  background: "linear-gradient(0deg, #FBF8FF 0%, #C7C7C7 100%)",
+  "&:hover, &:focus-within": {
+    boxShadow: theme.shadows.brandElevationBig,
+  },
+});
+
+const projectCardFooterStyle = css({
+  background: theme.colors.brandBackgroundProjectCardTextArea,
+  height: theme.spacing[17],
+  py: theme.spacing[5],
+  px: theme.spacing[7],
+});
+
+const projectNameAvatarStyle = css({
+  fontFamily: theme.fonts.manrope,
+  fontWeight: 200,
+  fontSize: 360,
+  alignSelf: "center",
+  letterSpacing: "-0.05em",
```

Comment:
The font styles should come from `typography.brandThumbnailLargeDefault` (this token is now in `main`)

<img width="858" alt="Screenshot 2023-01-25 at 12 11 24" src="https://user-images.githubusercontent.com/825702/214512261-ce803f90-9cb6-4954-9793-384ab18592a7.png">